jupyter_matlab_proxy/app.py

```diff
@@ -28,7 +28,13 @@
     from .gui.static import js
     from .gui.static import media
 
-logging.basicConfig(level=logging.INFO)
+logLevel=logging.WARNING
```

Comment:
To preserve compatibility with upstream I recommend to keep the default log level as info and override with the env var in our system